Content.Server/Whitelist/WhitelistCommands.cs

```diff
@@ -103,7 +108,7 @@ public override async void Execute(IConsoleShell shell, string argStr, string[]
                 return;
             }
 
-            await db.RemoveFromWhitelistAsync(guid);
+            await _dbManager.RemoveFromWhitelistAsync(guid);
```

Comment:
There's one more `db` -> `_dbManager` rename that seems to have gotten missed when cherry picking. It's a couple lines up from here:
```
var isWhitelisted = await db.GetWhitelistStatusAsync(guid);
```

Once you make that switch, you should be able to also remove the second set of these variables at new line 93, since they're no longer being used:

```
var db = IoCManager.Resolve<IServerDbManager>();
var loc = IoCManager.Resolve<IPlayerLocator>();
```


- upstream commit: `69b3e355e43f235e088c504952addef6b5598def`
- cherry pick commit from this PR: `e5cef17c82c42f4ac2deb6f529cc1f5391e902e3`